.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Self checklist**
+
+ - [ ] This issue hasn't been already created
+ - [ ] Everything is sound and clear, all sections are filled-up
+ - [ ] Relevant labels have been added
+
+# Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+# Describe the solution you'd like
```

Comment:
I would add this description on top

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/5090b071922f322cee876f9d6288b7279514d3b2